packages/api-runner/lib/fetchers/getPools.js

```diff
@@ -163,22 +170,39 @@ const combineTokenPricesData = (_pools, tokenPriceData, defaultTokenPriceUsd) =>
   const pools = cloneDeep(_pools)
 
   pools.forEach((pool) => {
+    // Add for token faucet drip tokens
+    Object.values(pool.tokenFaucets).forEach((tokenFaucet) => {
+      addTokenTotalUsdValue(tokenFaucet.dripToken, tokenPriceData, defaultTokenPriceUsd)
+    })
+
     // Add to all known tokens
-    Object.values(pool.tokens).forEach((token) =>
-      addTokenTotalUsdValue(token, tokenPriceData, defaultTokenPriceUsd)
-    )
+    Object.values(pool.tokens).forEach((token) => {
+      // This takes care of tokenFaucetDripTokens:
+      if (token?.length > 0) {
```

Comment:
```suggestion
      if (Array.isArray(token)) {
```

Just a little clearer